frontend/src/js/controllers/changePwdCtrl.js

```diff
@@ -7,67 +7,87 @@
         .module('evalai')
         .controller('ChangePwdCtrl', ChangePwdCtrl);
 
-    ChangePwdCtrl.$inject = ['utilities', 'validationSvc', '$state', '$http', '$rootScope'];
+    ChangePwdCtrl.$inject = ['utilities', '$state', '$http', '$rootScope'];
 
-    function ChangePwdCtrl(utilities, validationSvc, $state, $http, $rootScope) {
+    function ChangePwdCtrl(utilities, $state, $http, $rootScope) {
         var vm = this;
         var userKey = utilities.getData('userKey');
         vm.wrnMsg = {};
         vm.isValid = {};
+        vm.user = {};
+        vm.isFormError = false;
 
+        vm.changepassContainer = angular.element('.change-passowrd-card');
+
+        vm.startLoader = function(msg) {
+            $rootScope.isLoader = true;
+            $rootScope.loaderTitle = msg;
+            vm.changepassContainer.addClass('low-screen');
+        }
+
+        // stop loader
+        vm.stopLoader = function() {
+            $rootScope.isLoader = false;
+            $rootScope.loaderTitle = '';
+            vm.changepassContainer.removeClass('low-screen');
+        }
         // function to change password
-        vm.changePassword = function() {
-            vm.loginContainer = angular.element('.change-passowrd-card');
+        vm.changePassword = function(resetconfirmFormValid) {
+          if(resetconfirmFormValid){
+
 
+            vm.startLoader("Changing Your Password");
             var parameters = {};
             parameters.url = 'auth/password/change/';
             parameters.method = 'POST';
             parameters.data = {
                 "old_password": vm.user.old_password,
                 "new_password1": vm.user.new_password1,
                 "new_password2": vm.user.new_password2,
+                "uid": $state.params.user_id,
             }
-            parameters.token = userKey;
-            var passwordCheck = validationSvc.password_check(parameters.data.new_password1, parameters.data.new_password2);
-            if (passwordCheck.status) {
                 parameters.callback = {
```

Comment:
add this line before this
`parameters.token = userKey;`
Since no token is passing that causes 401 error.